pkgs/by-name/em/empire-compiler/package.nix

```diff
@@ -0,0 +1,39 @@
+{
+  fetchFromGitHub,
+  buildDotnetModule,
+  dotnetCorePackages,
+  lib,
+}:
+buildDotnetModule rec {
+  pname = "empire-compiler";
+  version = "0.3.3";
+
+  src = fetchFromGitHub {
+    owner = "bc-security";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "sha256-1SzP3oopmYy2Xv0CFxID4lSVZ65/MARd1O0w2zpdeyc=";
+  };
+
+  postPatch = ''
+    substituteInPlace EmpireCompiler/EmpireCompiler.csproj \
+      --replace-fail 'net6.0' 'net9.0'
+  '';
+
+  dotnet-sdk = dotnetCorePackages.sdk_9_0;
+  dotnet-runtime = dotnetCorePackages.aspnetcore_9_0;
+  nugetDeps = ./deps.json;
+
+  projectFile = "EmpireCompiler/EmpireCompiler.csproj";
+
+  meta = with lib; {
+    homepage = "https://github.com/BC-SECURITY/Empire-Compiler";
+    description = "The Empire Compiler started as the compiler of the Covenant project. It has since evolved to be an integral part of the Empire framework, aiding in the dynamic compilation and obfuscation of payloads.";
+    license = licenses.mit;
+    platforms = platforms.linux;
+    maintainers = with lib.maintainers; [
+      fzakaria
+      vrose
```

Comment:
@vinnybod are you okay with being added in? Need confirmation.

Reply:
Yes, thank you